5calls/app/src/main/java/org/a5calls/android/a5calls/controller/MainActivity.java

```diff
@@ -195,6 +198,17 @@ public void onJsonError() {
             public void onIssuesReceived(String locationName, List<Issue> issues) {
                 getSupportActionBar().setTitle(String.format(getResources().getString(
                         R.string.title_main), locationName));
+                // If this is the first time we've set issues ever, add all the contacts and
+                // issue IDs to the database to keep track of things the user has already called
+                // about. This allows upgrade from app version 0.07 without "unknown" contacts or
+                // issues in stats.
+                // Note: If the user has made calls in other locations, then we won't be saving
+                // the issues and contacts to the DB from those locations. However, that's probably
+                // OK.
```

Comment:
We should probably have a note to remove this code when none of our users are on version .7 anymore.